logzero/__init__.py

```diff
@@ -431,6 +426,37 @@ def logfile(filename, formatter=None, mode='a', maxBytes=0, backupCount=0, encod
         logger.addHandler(rotating_filehandler)
 
 
+def remove_internal_loggers(logger_to_update, disableStderrLogger=True):
+    """
+    Remove the internal loggers (e.g. stderr logger and file logger) from the specific logger
+    :param logger_to_update: the logger to remove internal loggers from
+    :param disableStderrLogger: should the default stderr logger be disabled? defaults to True
+    """
+    for handler in list(logger_to_update.handlers):
+        if hasattr(handler, LOGZERO_INTERNAL_LOGGER_ATTR):
+            if isinstance(handler, RotatingFileHandler):
+                logger_to_update.removeHandler(handler)
+            elif isinstance(handler, logging.StreamHandler) and disableStderrLogger:
+                logger_to_update.removeHandler(handler)
```

Comment:
should this also remove the syslog handler?

Reply:
Sure, that makes sense!